### PR TITLE
detail page past interviews

### PR DESCRIPTION
**Pull request by praveen021097**

### CODE REVIEW

File: src/Pages/BookInterviews/BookOneOnOne.tsx [hunk 1]
```diff
@@ -1,101 +1,163 @@
...
+  > = useDispatch();
+  const adminListDispatch: Dispatch<
+    | adminLIstByCategorySuccess
+    | adminLIstByCategoryFailure
+    | adminLIstByCategoryLoading
+  > = useDispatch();
+  useEffect(() => {
+    getAllCategoryDataService()(categoryDispatch);
+  }, []);
+  console.log("cate", categoryType);
```
Comment (sivagurubilli) [inline src/Pages/BookInterviews/BookOneOnOne.tsx:43]: remove console.log

Comment (sivagurubilli) [reply src/Pages/BookInterviews/BookOneOnOne.tsx:43]: remove console.log

File: src/Redux/CategoryReducer/Reducer.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { ActionTypes } from "./ActionTypes";
+import { Action } from "./Action";
+
```
Comment (sivagurubilli) [inline src/Redux/CategoryReducer/Reducer.ts:3]: remove spaces

File: src/Services/AuthService.tsx [hunk 3]
```diff
@@ -7,8 +7,13 @@ import { LoginData } from "../Pages/Login/LoginUser";
 
 export const loginService =(payload:LoginData)=>(dispatch:Dispatch<isLoginSuccess|isLoginFailure>):Promise<void | ActionTypes>=>{
 
+<<<<<<< Updated upstream
```
Comment (sivagurubilli) [inline src/Services/AuthService.tsx:10]: remove merge conflicts

**Review by pintu8328 — APPROVED** [review]

**Review by Shivamyadav07 — APPROVED** [review]

File: src/Services/AuthService.tsx (hunk 3 above)
Comment (praveen021097) [reply src/Services/AuthService.tsx]: resolved

File: src/Pages/PastEvents/PastEvents.tsx [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+import Header from '../../Components/CommonComponents/Header'
+import Navbar from '../../Components/Navbar/Navbar'
+import React, { useEffect } from 'react'
+import { useDispatch, useSelector } from "react-redux";
+import { RootState } from "../../Redux/store";
+import { convertTimeFormat } from "../../utils/index";
+import { Action } from "../../Redux/PastInterviewReducer/Action";
+import { Dispatch } from "redux";
+import { getAllPastInterviewService } from "../../Services/UserSideServices/GetAllPastInterviewServices/GetAllPastInterviewService";
+const PastEvents = () => {
```
Comment (mangesh0712) [inline src/Pages/PastEvents/PastEvents.tsx:27]: use typescript to define comp

File: src/Redux/AdminListByCategoryReducer/Action.ts [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    type:ActionTypes.GET_ALL_ADMIN_LIST_BY_CATEGORY_SUCCESS,
+    payload:any
+}
+
+export interface adminLIstByCategoryLoading{
+    type:ActionTypes.GET_ALL_ADMIN_LIST_BY_CATEGORY_REQUEST,
+    payload:true
+}
+
+export interface adminLIstByCategoryFailure{
```
Comment (mangesh0712) [inline src/Redux/AdminListByCategoryReducer/Action.ts:13]: dminListByCategoryFailure

**Pull request merged**